lametro/models.py

```diff
@@ -487,6 +487,26 @@ class LAMetroEvent(Event, LiveMediaMixin, SourcesMixin):
     class Meta:
         proxy = True
 
+    @classmethod
+    def most_recent_past_meetings(cls):
+        '''
+        Returns meetings in the current month that have occured in the past
+        two weeks.
+        '''
+        current_month = timezone.now().month
+        two_weeks_ago = timezone.now() - timedelta(weeks=2)
+
+        meetings_in_past_two_weeks = cls.objects.filter(
+            start_time__month=current_month).filter(
+            start_time__gte=two_weeks_ago)
```

Comment:
You can combine these keyword arguments into a single `filter()` call, like this: `filter(kwarg_1=foo, kwarg_2=bar)`